bindings/capacitorjs/src/definitions.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type { EventType, AccountManagerOptions } from './types';
+export interface IotaWalletMobileTypes {
+    initLogger(path: string): Promise<void>;
```

Comment:
In the nodeJS bindings this represents the `LoggerConfig` that is then JSON.stringify(). Does it need something similar here instead of `path`?

Reply:
This part needs to be finished on the Java side: https://github.com/iotaledger/wallet.rs/issues/1790